### PR TITLE
clr managed heap document improvements

### PR DESCRIPTION
**Pull request by networkfusion**

### CODE REVIEW

File: docs/ChibiOS HAL/clr-managed-heap.md [hunk 1]
```diff
@@ -2,38 +2,39 @@
...
 
-This document describes how the CLR manged heap is defined in a ChibiOS target.
+This document describes how the CLR manged heap is defined as a ChibiOS target.
 
-These configurations are located in two linker files: 
-- [rules.ld](../../targets/CMSIS-OS/ChibiOS/common/rules.ld) (common to all ChibiOS targets **except** the ones with F7 series that have to use [rules_STM32F7xx.ld](../../targets/CMSIS-OS/ChibiOS/common/rules_STM32F7xx.ld) instead)
-- the target linker file provided for nanoCLR at the target board folder, e.g. [STM32F091xC.ld](../../targets/CMSIS-OS/ChibiOS/ST_NUCLEO_F091RC/nanoCLR/STM32F091xC.ld)
+For STM32 based devices:
+The configurations are spread between two linker files: 
+- [rules.ld](../../targets/CMSIS-OS/ChibiOS/common/rules.ld) (which is common to all STM32 based ChibiOS targets **except** the F7 series that use [rules_STM32F7xx.ld](../../targets/CMSIS-OS/ChibiOS/common/rules_STM32F7xx.ld) instead)
```
Comment (networkfusion) [inline docs/ChibiOS HAL/clr-managed-heap.md]: Actually I don't think this is true any more, the F7 actually defines the differences within its own ld.

Comment (josesimoes) [inline docs/ChibiOS HAL/clr-managed-heap.md]: I agree!

Comment (josesimoes) [inline docs/ChibiOS HAL/clr-managed-heap.md]: I believe that you are correct. Care to update the bit above so this can be merged? 😉

Comment (piwi1263) [inline docs/ChibiOS HAL/clr-managed-heap.md]: Who ever made the change, should check it's spellchecker. It has some bugs, line 26 and 32 have some new words I didn't know before or and most likely the words exaple (line 26) and manged (line 32) are waiting to be corrected.

Comment (networkfusion) [inline docs/ChibiOS HAL/clr-managed-heap.md]: @piwi1263 done (I didn't have a spell checker installed, I have now  :-p) typing too quickly for my own good!

**Pull request merged**